internal/operator/cluster/cluster.go

```diff
@@ -234,8 +239,16 @@ func AddClusterBootstrap(clientset kubeapi.Interface, cluster *crv1.Pgcluster) e
 	ctx := context.TODO()
 	namespace := cluster.GetNamespace()
 
-	if err := operator.CreatePGHAConfigMap(clientset, cluster, namespace); err != nil &&
-		!kerrors.IsAlreadyExists(err) {
+	var err error
+
+	if err = operator.CreatePGHAConfigMap(clientset, cluster,
+		namespace); kerrors.IsAlreadyExists(err) {
+		log.Debugf("found existing pgha ConfigMap for cluster %s, setting init flag to 'true'",
+			cluster.GetName())
```

Comment:
I would probably put this one at the Info level.

Reply:
Changed to `Info`.